### PR TITLE
fix(SelectWithLogo): fix qa props types

### PR DESCRIPTION
**Pull request by castrix**
fix types

### CODE REVIEW

File: src/components/SelectWithLogo/SelectWithLogo.tsx [hunk 1]
```diff
@@ -150,7 +150,7 @@ const OptionTemplateWithDataQa = (props: OptionTemplateWithDataQaProps) => {
       {...innerProps}
       ref={innerRef}
       data-qa-id={dataQa}
-      data-qa-props={dataQaProps}
+      data-qa-props={dataQaProps ? dataQaProps(props) : `value:${props.data.value}`}
```
Comment (rayhan-ab) [inline src/components/SelectWithLogo/SelectWithLogo.tsx:153]: why do we update data-qa-props logic?

**Comment by rayhan-ab** [comment]
please update the changelog

File: src/components/SelectWithLogo/SelectWithLogo.tsx (hunk 1 above)
Comment (castrix) [reply src/components/SelectWithLogo/SelectWithLogo.tsx:153]: the `data-qa-props` in the original component was `value:${props.data.value}` and when I check the output, it's iterating all the options value, so I return it to the original logic and add the props for customization

**Review by rayhan-ab — APPROVED** [review]

**Comment by rayhan-ab** [comment]
please fix the conflict and you can merge

**Comment by castrix** [comment]
> please fix the conflict and you can merge

got it, but I'll need two approvals if I'm the one who merge it

**Comment by rayhan-ab** [comment]
i think it only need 1 approval

**Comment by rayhan-ab** [comment]
or just fix the conflict, i'll merge

**Pull request merged**